client/src/styles/global.scss

```diff
@@ -135,7 +136,7 @@ $primary-color: #112f4e;
       float: left; // allows vertical centering on logo and text
       margin-right: 0.5em; // space between logo and text
```

Comment:
When are we electing to you use em over rems? I've been using rems pretty much throughout

Reply:
Actually, it's a bug. I'll flip them all to `rem`.

Reply:
fixed in latest.